src/dipdup/cli.py

```
@@ -562,7 +580,52 @@ async def hasura_configure(ctx: click.Context, force: bool) -> None:
 @_cli_wrapper
 async def schema(ctx: click.Context) -> None:
     """Commands to manage database schema."""
-    pass
+    from pathlib import Path
+
+    from dipdup.aerich import create_aerich_command
+
+    if '--help' in sys.argv:
+        return
+
+    config: DipDupConfig = ctx.obj.config
+
+    if ctx.invoked_subcommand in AERICH_CMDS:
+        if not Path(config.database.migrations_dir).exists():
+            echo(
+                f"""Database migrations is not initialize at {config.database.migrations_dir}.
+                 Run `dipdup schema init` or just run `dipdup run` and it'll be initialized automatically."""
+            )
+            raise click.Abort()
+
+        aerich_command = await create_aerich_command(config)
+        await aerich_command.init()
+
+        ctx.obj['command'] = aerich_command
+
+
+# Wrapper function to approve schema after upgrade and downgrade
+# Since upgrade and downgrade are meant to be intentionally run by the user, it'll be confusing for the user to run approve just after them
+def _approve_schema_after(command: click.Command) -> click.Command:
+    @click.pass_context
+    def wrapper(ctx: click.Context, /, *args: Any, **kwargs: Any) -> None:
+        ctx.invoke(command, *args, **kwargs)
+        # TODO: don't call approve if no upgrades/downgrades happened
+        ctx.invoke(schema_approve)
+
+    return click.Command(
+        name=command.name,
+        callback=wrapper,
+        params=command.params,
+        help=command.help,
+        short_help=command.short_help,
+    )
+
+
+schema.add_command(aerich_cli.commands['history'])
```

Please wrap this with try except to silently fail when aerich is not installed.